fe/fe-core/src/main/java/org/apache/doris/datasource/InternalDataSource.java

```diff
@@ -198,7 +198,7 @@
  * There is only one internal data source in a cluster. And its id is always 0.
  */
 public class InternalDataSource implements DataSourceIf {
-    public static final String INTERNAL_DS_NAME = "__internal";
```

Comment:
Need #10213 to be merged first. Underscores will case authentication exception. I will remove this after #10213 is merged.